ec2utils/ec2uploadimg/python-ec2uploadimg.spec

```diff
@@ -18,7 +18,7 @@
 
 %define upstream_name ec2uploadimg
 Name:           python-ec2uploadimg
-Version:        2.0.0
+Version:        2.1.0
```

Comment:
semver.org reads "increment the minor version when you add functionality in a backwards-compatible manner". Which I believe I did.